graphql/schema.graphql

```diff
@@ -815,6 +829,7 @@ type PublishPublicKeyPayload {
 }
 
 type Query {
+  GetBindingsQuery (name: String!, version: String = "latest"): PackageVersion
```

Comment:
This seems wrong?

Reply:
`GetBindingsQuery` is not a field in the `Query` ObjectType

Reply:
Oops! I'm guessing I forgot to remove that after experimenting before :sweat_smile: